integr8ly/integreatly-operator
# MGDAPI-3426 bump 3scale operator to 0.8.3

# Issue link
https://issues.redhat.com/browse/MGDAPI-3426

# What
Bump to 3scale operator to version 0.8.3

# Verification steps
- run initial install and confirm installation succeeds 
- run upgrade from RHOAM 1.18.2 -> mock 1.19 containing changes from this PR and confirm upgrade succeeds (possibly use WLWA  to meassure downtime)
